charts/fluent-bit/Chart.yaml

```diff
@@ -22,5 +22,5 @@ maintainers:
     email: steve.hipwell@gmail.com
 annotations:
   artifacthub.io/changes: |
-    - kind: fixed
-      description: "Fixed helm tests."
+    - kind: changed
+      description: "Added local internalTrafficPolicy for daemonsets"
```

Comment:
Could you please re-add the line ending?

Reply:
@stevehipwell not sure how i did that. resolved the conflict via github on my phone, i guess that'll teach me?